packages/core/admin/admin/src/components/FormInputs/DateTime.tsx

```diff
@@ -11,7 +11,7 @@ import { InputProps } from './types';
 const DateTimeInput = forwardRef<HTMLInputElement, InputProps>(
   ({ name, required, label, hint, labelAction, ...props }, ref) => {
     const { formatMessage } = useIntl();
-    const field = useField<Date>(name);
+    const field = useField(name);
```

Comment:
How about `Date | null`?

Reply:
yeah, I used the same instruction we used in the Date.tsx but for sure I can change both lines